modules/monitoring-determining-why-prometheus-is-consuming-disk-space.adoc

```diff
@@ -6,19 +6,23 @@
 [id="determining-why-prometheus-is-consuming-disk-space_{context}"]
 = Determining why Prometheus is consuming a lot of disk space
 
-Labels are used to define attributes for metrics in the form of key-value pairs. A unique time series is created for every key-value pair. Adding many values to labels can result in an exponential increase in the number of time series created. This can impact Prometheus performance and can consume a lot of disk space.
+Developers can create labels to define attributes for metrics in the form of key-value pairs. The number of potential key-value pairs corresponds to the number of possible values for an attribute. An attribute that has an unlimited number of potential values is called an unbound attribute. For example, a `customer_id` attribute is unbound because it has an infinite number of possible values.
 
-When Prometheus consumes a lot of disk:
+.The impact of using many unbound attributes in labels
```

Comment:
I don't think we normally use this type of heading, especially in a procedural module, which has a pretty set template of Prerequisites/Procedure headings.

Also, it makes it look like everything until you hit "Prerequisites" falls under this heading, when I think it's really intended to be just the first paragraph.

I'd consider putting this paragraph in a note/warning here.

Reply:
@bergerhoffer thank you for your review and for highlighting this.

On reflection I have just removed the dot heading. That paragraph is the main point of the introduction to this section and so I would prefer it not to be in a note. I included the heading to improve scannability, but I completely see your point about how it looks like the heading also relates to the subsequent content.